canonical-websites/www.ubuntu.com
# Create a standard include for landing page forms

## Done

* Created a generalized form include for landing pages with two parameters: `id` and `returnURL`
* Use it and replace hard coded form on `/engage/multi-cloud-guide`

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Ensure the form at `/engage/multi-cloud-guide` is taking you to the right download
